File: dodam-infra/src/main/java/b1nd/dodaminfra/token/JwtProperties.java
package b1nd.dodaminfra.token;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Getter
@Setter
@Configuration
@ConfigurationProperties("app.jwt")
class JwtProperties {

    private String issuer;
    private String tokenServer;

}

File: services/service-auth/src/main/kotlin/com/b1nd/dodamdodam/auth/infrastructure/security/jwt/properties/JwtProperties.kt
package com.b1nd.dodamdodam.auth.infrastructure.security.jwt.properties

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("app.jwt")
data class JwtProperties(
    val privateKey: String,
    val publicKey: String,
    val accessExpireSeconds: Long,
    val refreshExpireSeconds: Long,
    val issuer: String
)
